evasquez26/angular4-startup
# S3-ONE-124 Accept/reject role: Teacher

## Story / Task

see [ONE-124](https://bixlabs.atlassian.net/browse/ONE-124)


## What this PR does?

Adds functionality to accept or reject a role request type teacher


### How to test / reproduce

- Login to the app
- Verify you have the role admin in any organization
- Go to Admin menu. You'll see a list of role requests you can accept or reject
- Click accept. You'll see a confirmation message. Confirm.
- The request status should be displayed as closed.
- Get a new role request and click reject. You'll see a confirmation message. Confirm.
-  The request status should be displayed as closed. And you are prompted to add a comment. Write a comment and click send.
- The requester should see the role request as closed and should be able to see the comment.



### Related PRs
N/A

## Migrations

 NO

Thanks.
